api.planx.uk/modules/send/utils/exportZip.ts

```diff
@@ -134,9 +134,13 @@ export async function buildSubmissionExportZip({
   });
 
   // add an optional GeoJSON file to zip
-  const geojson = passport?.data?.["property.boundary.site"];
+  const geojson: GeoJSON.Feature | undefined =
+    passport?.data?.["property.boundary.site"];
   if (geojson) {
-    if (userAction) geojson["properties"]["planx_user_action"] = userAction;
+    if (userAction) {
+      geojson["properties"] ??= {};
```

Comment:
Without this line I can now recreate the bug with the enriched mock data - 

![image](https://github.com/theopensystemslab/planx-new/assets/20502206/74d5552d-1581-441b-890d-2354018ddbcf)